examples/endian_swapper/cosim/Makefile

```diff
@@ -1,30 +1,32 @@
-include $(shell cocotb-config --makefiles)/Makefile.inc
-include $(shell cocotb-config --makefiles)/Makefile.pylib
+PYTHON_BIN ?= $(realpath $(shell cocotb-config --python-bin))
 
-PYTHON_LIBDIR ?= /usr/lib64
-SWIG ?= swig
-CC=gcc
-ifeq ($(ARCH),i686)
-	CC += -m32
+PYTHON_LIBDIR := $(shell $(PYTHON_BIN) -c 'from distutils import sysconfig; print( sysconfig.get_config_var("LIBDIR") )')
+ifeq ($(PYTHON_LIBDIR),None)
+    PYTHON_LIBDIR := $(shell dirname $(PYTHON_BIN))/libs
 endif
```

Comment:
I think we have similar code in at least two places now. Can we add a --python-libdir and --python-includedir option to cocotb-config instead?

Reply:
Only here in the makefiles (pylib will be removed). We will have only (some of it) in `_build_libs.py` as I can see.

Reply:
But this is can be done. Possible useful and much easier to control from Python any fixes.

Reply:
I don't think there's much value in doing this. A better use of time would be for this example to show how to use setuptools to build an extension, and have it invoke `$(PYTHON_BIN) setup.py build_ext` or similar

Reply:
Long run I would like to do what @eric-wieser suggests.

Reply:
Don't worry if it's used only once.